config-staging.json

```diff
@@ -20,7 +20,8 @@
       "shortName": "Ethereum",
       "supportedBy": [
         "DSC_DATA",
-        "DSC_TRANSACTIONS"
+        "DSC_TRANSACTIONS",
+        "DSC_ARTIFACTS"
```

Comment:
Can we keep these in alphabetical order ples, just in case the list gets larger it's easier to maintain?